README.md

```diff
@@ -1 +1,16 @@
-# life-at-ubc
\ No newline at end of file
+# Life @ UBC
+
+### Installation (if you don't have these already)
+- Node from https://nodejs.org/en/
+- Git https://git-scm.com/downloads
+- VS Code (Optional) https://code.visualstudio.com/
+
+#### Client
+- `cd client`
+- `npm install` to install all dependencies 
+- `npm start` to build & start the React app
+
+#### Server
+- `cd server`
+- `npm install` in the root directory to install all dependencies
+- `npm start` to start the Node.js server
```

Comment:
Should we set up a script in the `package.json` file so that we can run a single command that will launch the client and server together? That way we won't have to open 2 terminals each time. I'm open to hear what other devs prefer as well!

We could try something like [Concurrently](https://www.npmjs.com/package/concurrently).

Reply:
We can probably create a new issue for that, and someone can add it, but I think for this initial set up this is fine.

Reply:
Sounds good, resolving this comment.